ui/app/src/utils/i18n-legacy.ts

```diff
@@ -154,6 +154,25 @@ const groupsAdminMessages = defineMessages({
   }
 });
 
+const settingsMessages = defineMessages({
```

Comment:
Rename to profileSettingsMessages. Top var and ids.

Reply:
Done